pkg/webui/console/store/middleware/logics/init.js

```diff
@@ -49,7 +49,6 @@ const consoleAppLogic = createRequestLogic({
       dispatch(user.getUserRightsSuccess(rights))
     } catch (error) {
       dispatch(user.getUserRightsFailure())
-      dispatch(user.logout())
```

Comment:
mb not in this pr, but i guess we dont want to just ignore  `getUserRightsFailure` action. otherwise the user cant login without any error ui

Reply:
Right. This will be caught by refactoring the `withAuth` component to check for these errors in the next PR. For now, I'll just rethrow the error so it will be caught by the error boundary.